reports/glides_validator.livemd

```diff
@@ -22,90 +24,553 @@ Mix.install(
 )
 ```
 
-## Load Data
+## Instructions
 
 ### README
 
 This report provides an implementation of https://www.notion.so/mbta-downtown-crossing/Rethinking-Prediction-Accuracy-for-Glides-e99561127b01490689135ab6b70cd33c
 
+### Generate a report
+
+1. Click "Evaluate" on the code cell in the [**Options for Report**](#options-for-report) section.
+2. Scroll to the bottom of the code cell and choose settings for your report.
+3. Jump to the [**Results**](#results) section and click "Evaluate" on its code cell.
+4. Scroll to the bottom of the code cell.
+5. Wait around 30 sec to see results. Takes longer the first time it runs. Click "Export as CSV" to save results to your Downloads folder.
+
+If you want to generate another report with different settings, simply change the settings, scroll back down to [**Results**](#results), and click "Evaluate" again.
+
+## Hardcoded stop data
+
+**Feel free to collapse this section.**
+
+It needs to come before the section below, but is not important.
+
+```elixir
+defmodule GlidesReport.Terminals do
+  @moduledoc "Harcoded data on terminal stops, and fns to query it."
+
+  @terminals [
```

Comment:
meta commentary: I wish we had a central source of truth for this information. It's hardcoded into prediction analyzer as well.

Reply:
In theory it could be computed from a bunch of V3 API queries, but that's just a lot more work than hardcoding it unfortunately. The fact that some shuttle route patterns are marked as canonical right alongside the "true" canonical route pattern certainly doesn't help...